pspgl/glut.c

```diff
@@ -5,6 +5,7 @@
 #include <pspkerneltypes.h>
 #include <pspctrl.h>
 #include <psploadexec.h>
+#include "glut_shapes.c" // Disable this if you don't want GLUT shapes support
```

Comment:
`.c` files are not really intended for including ;)

Reply:
Remove this include, and instead, in `/pspgl/Makefile`, 
append `glut_shapes.c` to glut target, like this:

```makefile
libglut.a_OBJS = \
	glut.o
        glut_shapes.o
```